internal/tfengine/tfengine.go

```diff
@@ -182,11 +182,9 @@ func dumpTemplate(conf *Config, pwd, cacheDir, outputPath string, ti *templateIn
 	// Pass through specified keys that should be validated against the
 	// schema.
 	for _, k := range ti.Passthrough {
-		v, ok := conf.Data[k]
-		if !ok {
-			return fmt.Errorf("did not find key %q to pass through to child template", k)
+		if v, ok := conf.Data[k]; ok {
```

Comment:
merging issue?

Reply:
No, see the description. It doesn't work with error checking because the "state_path_prefix field" isn't always passed in. I was thinking it might be simpler to just let schemas in children be responsible for throwing errors if fields aren't passed.

Reply:
I see. Can you add a comment for that in the code as well?